src/OrbitGl/LiveFunctionsController.cpp

```diff
@@ -35,9 +35,8 @@ std::pair<uint64_t, uint64_t> ComputeMinMaxTime(
 uint64_t AbsDiff(uint64_t a, uint64_t b) {
   if (a > b) {
     return a - b;
-  } else {
-    return b - a;
   }
+  return b - a;
```

Comment:
Nit: This is unrelated and should be a separate commit.

Reply:
Done.